changelog.md

```diff
@@ -64,6 +64,7 @@
 - [#3337](https://github.com/ignite/cli/pull/3337) Register`nodeservice` gRPC in `app.go` template.
 - [#3434](https://github.com/ignite/cli/pull/3434) Detect app wiring implementation.
 - [#3445](https://github.com/ignite/cli/pull/3445) refactor: replace `github.com/ghodss/yaml` with `sigs.k8s.io/yaml`
+- [#3444](https://github.com/ignite/cli/pull/3444) Add support for ICS chains in ts-client generation
```

Comment:
The entry should be moved to the top of the `changelog.md` file into a new **Changes** sub section under **Unreleased** 🙏